src/classes/STG_PanelDataImportAdvancedMapping_CTRL.cls

```diff
@@ -39,7 +39,7 @@ public with sharing class STG_PanelDataImportAdvancedMapping_CTRL extends STG_Pa
     /*********************************************************************************************************
     * @description URL of the Advanced Mapping Custom UI aura component container
     */
-    public static final String COMPONENT_URL = '/lightning/cmp/c__BDI_ManageAdvancedMapping';
+    public static String componentUrl = '/lightning/cmp/npsp__BDI_ManageAdvancedMapping';
```

Comment:
Since this is in Apex, how about something like the following this way you're not hardcoding a namespace that only works in a namespaced org.
```suggestion
    public static String componentUrl = '/lightning/cmp/' + 
        (String.IsBlank(UTIL_Namespace.getNamespace()) ? 'c' : UTIL_Namespace.getNamespace()) +
        '__BDI_ManageAdvancedMapping';
```